### PR TITLE
Update ScenegraphLayer to use GLTFLoader

### PR DESCRIPTION
**Pull request by ibgreen**
# For #3282 

#### Background
- Allows GLTF files to be loaded by the application and passed in.

- Tested in the layer browser
- yarn.lock and package.json changes in separate PR here: https://github.com/uber/deck.gl/pull/3287/files

### CODE REVIEW

File: modules/mesh-layers/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "@deck.gl/core": "^7.0.0"
   },
   "dependencies": {
+    "@luma.gl/addons": "^7.2.0-alpha.1",
```
Comment (georgios-uber) [inline modules/mesh-layers/package.json]: Can this be 7.10? I don't like the alpha part.

File: modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.js [hunk 2]
```diff
@@ -43,7 +44,8 @@ const defaultProps = {
     return fetch(url).then(response => response.json());
   },
 
-  getScene: scenegraph => (scenegraph && scenegraph.scenes ? scenegraph.scenes[0] : scenegraph),
+  getScene: scenegraph =>
+    scenegraph && scenegraph.scenes ? scenegraph.scenes[scenegraph.scene || 0] : scenegraph,
```
Comment (georgios-uber) [inline modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.js]: 👍

File: test/modules/react/utils/evaluate-children.spec.js [hunk 3]
```diff
@@ -12,6 +12,7 @@ const TEST_CASES = [
   },
   {
     title: 'function child',
+    // eslint-disable-next-line react/display-name
```
Comment (georgios-uber) [inline test/modules/react/utils/evaluate-children.spec.js]: Why is this needed here?

File: modules/mesh-layers/package.json (hunk 1 above)
Comment (georgios-uber) [reply modules/mesh-layers/package.json]: It might cause 2 versions of luma to be installed.

Comment (ibgreen) [reply modules/mesh-layers/package.json]: Core is also including `alpha.1`
